### PR TITLE
feat: APY evolution - add sample youtube link, clean up lint issues

### PR DESCRIPTION
**Pull request by mrbasado**

### CODE REVIEW

**Comment by mrbasado** [comment]
no youtube embed for now until we get a link

File: src/components-v2/VaultApyInformation/index.tsx [hunk 1]
```diff
@@ -301,10 +301,10 @@ const VaultApyInformation = ({ open, onClose, boost, vault, projectedBoost }: Pr
               <Grid item>Rewards earned by our strategies on your vault deposits</Grid>
             </Grid>
             {yieldSourcesApyList.map((yieldSource) => (
-              <>
+              <Box key={`yieldSourcesApyList-${yieldSource.name}`}>
```
Comment (axejintao) [inline src/components-v2/VaultApyInformation/index.tsx:304]: should L164 be any? best to use type inference or a proper type if we can

File: src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx [hunk 2]
```diff
@@ -62,15 +69,24 @@ const YieldBearingRewards = observer(({ open, onModalClose }: YieldBearingReward
       aria-describedby="alert-dialog-description"
     >
       <DialogTitle id="alert-dialog-title" className={classes.title}>
-        <Typography variant="h6" color="primary" className={classes.titleText}>
-          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" /> Yield-Bearing Rewards
-        </Typography>
+        <Box className={classes.titleText}>
+          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" />
```
Comment (axejintao) [inline src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:73]: this needs an `alt` field

File: src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx [hunk 3]
```diff
@@ -62,15 +69,24 @@ const YieldBearingRewards = observer(({ open, onModalClose }: YieldBearingReward
...
+            Yield-Bearing Rewards
+          </Typography>
+        </Box>
         <IconButton aria-label="close" className={classes.closeButton} onClick={onModalClose}>
           <CloseIcon />
         </IconButton>
       </DialogTitle>
       <DialogContent id="alert-dialog-description" className={classes.root}>
-        <Box className={classes.video}></Box>
+        {/* <Box className={classes.video}>
```
Comment (axejintao) [inline src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:83]: do we want this commented out?

Comment (mrbasado) [reply src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:83]: yeah will wants no embed till he makes the youtube video which is going to be latter than the release date for this

File: src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx (hunk 2 above)
Comment (mrbasado) [reply src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:73]: good catch

Comment (mrbasado) [reply src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:73]: wait did you mean to link something else? l73 has an alt

**Review by axejintao — APPROVED** [review]

File: src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx (hunk 2 above)
Comment (axejintao) [reply src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx:73]: no i am blind

**Pull request merged**